---
title: "Publications"
---

## Peer revied publications

* Markku Kuismin and Mikko J. Sillanpää. tvsfglasso: time-varying scale-free graphical lasso for network estimation from time-series data. *PLOS Computational Biology*, 21:e1013710, 2025. [https://doi.org/10.1371/journal.pcbi.1013710](https://doi.org/10.1371/journal.pcbi.1013710)

* Aapo. E. Korhonen, Olli Sarala, Tuomas Hautamäki, Markku Kuismin, and Mikko J. Sillanpää. HMFGraph: Novel Bayesian approach for recovering biological networks. *PLOS Computational Biology*, 21:e1013614, 2025. [https://doi.org/10.1371/journal.pcbi.1013614,](https://doi.org/10.1371/journal.pcbi.1013614)

* Tero Lähderanta, Lauri Lovén, Leena Ruha, Teemu Leppänen, Markku Kuismin, Ilkka Launonen, Susanna Pirttikangas,
Jukka Riekki, and Mikko J. Sillanpää. The intersection of
location-allocation, partitional clustering and model-based clustering techniques - a review. *Statistical Science*, 2024.

* Markku Kuismin and Mikko J. Sillanpää. Network hub detection using the entire solution path information. *GENETICS*, 2024. [https://doi.org/10.1093/genetics/iyae187](https://doi.org/10.1093/genetics/iyae187)

* Markku Kuismin. Using rejection sampling probability of acceptance as a measure of independence. *Journal of Computational and Graphical Statistics*, 1-17, 2024.
[https://doi.org/10.1080/10618600.2024.2388544](https://doi.org/10.1080/10618600.2024.2388544)

* Ville Happonen, Heikki Kröger, Markku Kuismin, and Reijo
Sund. Ankle fractures in finland: 118,929 operatively treated between 1987 and 2019. *Acta orthopaedica*, 93:327-333, 2022. [https://doi.org/10.2340/17453674.2022.2071](https://doi.org/10.2340/17453674.2022.2071)

* Markku Kuismin, Fatemeh Dodangeh, and Mikko J. Sillanpää.
Gap-com: general model selection criterion for sparse undirected gene networks with nontrivial community structure. *G3 (Bethesda)*, 2021. [https://doi.org/10.1093/g3journal/jkab437](https://doi.org/10.1093/g3journal/jkab437)

* Dilan Saatoglu, Alina K. Niskanen, Markku Kuismin, Peter S. Ranke, Ingerid J. Hagen, Yimen G. Araya-Ajoy,
Thomas Kvalnes, Henrik Pärn, Bernt Rønning, Thor Harald Ringsby, Bernt-Erik Sæther, Arild Husby, Mikko J. Sillanpää, and Henrik Jensen. Dispersal in a house sparrow metapopulation: An integrative case study of genetic assignment calibrated with ecological data and pedigree information. Molecular Ecology, 30:4740–4756, 2021.
[https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.1608](https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.1608)

* Markku Kuismin and Mikko J. Sillanpää. MCPeSe: Monte Carlo
penalty selection for graphical lasso. Bioinformatics, 37:726-727, 2020. [https://doi.org/10.1093/bioinformatics/btaa734](https://doi.org/10.1093/bioinformatics/btaa734) 

* Markku Kuismin, Dilan Saatoglu, Alina K. Niskanen, Henrik Jensen, and Mikko J. Sillanpää. Genetic assignment of
individuals to source populations using network estimation
tools. Methods in Ecology and Evolution, 11:333-344, 2019.
[https://doi.org/10.1111/2041-210X.13323](https://doi.org/10.1111/2041-210X.13323)

* Markku O. Kuismin and Mikko J. Sillanpää. Estimation of covariance and precision matrix, network structure,
and a view toward systems biology. Wiley Interdisciplinary Reviews: Computational Statistics, 9:e1415, 2017. [https://doi.org/10.1002/wics.1415](https://doi.org/10.1002/wics.1415)

* M. O. Kuismin, J. Ahlinder, and M. J. Sillanpää. CONE:
Community oriented network estimation is a versatile
framework for inferring population structure in large-scale sequencing data. G3 (Bethesda), 7:3359-3377, 2017.
[https://doi.org/10.1534/g3.117.300131](https://doi.org/10.1534/g3.117.300131)

* M. O. Kuismin, J. T. Kemppainen, and M. J. Sillanpää. Precision matrix estimation with ROPE. Journal
of Computational and Graphical Statistics, 26:682–694, 2017. [https://doi.org/10.1080/10618600.2016.1278002](https://doi.org/10.1080/10618600.2016.1278002)

* Markku Kuismin and Mikko J. Sillanpää. Use of wishart
prior and simple extensions for sparse precision matrix estimation. PLOS ONE, 11:e0148171, 2016. [https://doi.org/10.1371/journal.pone.0148171](https://doi.org/10.1371/journal.pone.0148171)

## Manuscripts

* Olli Sarala, Markku Kuismin, Tuomas Hautamäki, Aapo E. Korhonen and Mikko J. Sillanpää. Bayesian estimation of sparse partial correlation graphs with smooth temporal evolution.

* Tuomas Hautamäki, Aapo E. Korhonen, Olli Sarala, Markku Kuismin and Mikko J. Sillanpää. GHSCM: Efficient maximum a posteriori inference for biological networks with the graphical horseshoe prior.

* Markku Kuismin, Jon Ahdlinder, and Mikko J. Sillanpää. fast network based microbial source tracking method can detect low-level fecal pollution

## Other publications

* Markku Kuismin. Using the rejection sampling for finding tests. arXiv preprint arXiv:2509.10325, 2025. [https://arxiv.org/abs/2509.10325](https://arxiv.org/abs/2509.10325) 

* Leena Ruha, Tero Lähderanta, Lauri Lovén, Markku Kuismin, Teemu Leppänen, Jukka Riekki, and Mikko J. Sillanpää. Capacitated spatial clustering with multiple constraints and attributes. arXiv preprint arXiv:2010.06333, 2020. [https://arxiv.org/abs/2010.06333](https://arxiv.org/abs/2010.06333)